recipe/meta.yaml

```diff
@@ -22,12 +22,12 @@ requirements:
     - python
     - poetry-core 1.5
   run:
-    - markdown-it-py >=2.2.0,<3.0.0
-    - pygments >=2.13.0,<3.0.0
+    - markdown-it-py >=2.2.0
+    - pygments >=2.13.0
```

Comment:
`^2.13.0` means `>=2.13.0,<3.0.0`, see poetry's Caret requirements https://python-poetry.org/docs/dependency-specification/#caret-requirements
 https://github.com/Textualize/rich/blob/7f580bdcf07a3b269a0e786b6a3aa9c804f393cf/pyproject.toml#L32

Reply:
we discussed this with @psteyer and having a look at recent other discussions about upper bounds, I suggested to remove upper bounds (also above) see `Pinnings` in our knowledge base

Reply:
I removed the upper bounds after my conversation with @lorepirri . What is the best way to proceed on this @skupr-anaconda?

Reply:
I can't agree with all aspects of `Pinnings` in our knowledge base when it's related to upper bounds that were set by maintainers. Often it's a tradition but sometimes it's a warning that things go wrong and the maintainers explicitly say **don't** touch. The major version bump often says that there are breaking changes and without the upper bound all downstream or upstream packages will fail.
It's about of point of view: by the maintainer or by the package builder/distributor of the software.
Generally, I follow the upstream but sometimes I patch version's constraints if I tested that it works fine.

Reply:
I'm fine with reintroducing the upper constraints.